### PR TITLE
Add FNet Presets

### PR DESCRIPTION
**Pull request by abheesht17**
`f_net_base_en`: https://colab.research.google.com/drive/1VcLbisTI72yUhufLwxPmwGotNRMvhI4U?usp=sharing
`f_net_large_en`: https://colab.research.google.com/drive/1cHGAt1VUASCgMm_pHpvRIJHOrKVXsxqK?usp=sharing

### CODE REVIEW

File: keras_nlp/models/deberta_v3/deberta_v3_presets.py [hunk 1]
```diff
@@ -90,7 +90,7 @@
         },
         "preprocessor_config": {},
         "description": (
-            "Base size of DeBERTaV3. "
+            "Large size of DeBERTaV3. "
```
Comment (jbischof) [inline keras_nlp/models/deberta_v3/deberta_v3_presets.py:93]: Move to separate PR.

File: keras_nlp/models/f_net/f_net_tokenizer.py [hunk 2]
```diff
@@ -84,12 +88,53 @@ def __init__(self, proto, **kwargs):
...
 
     @classmethod
+    @format_docstring(names=", ".join(backbone_presets))
     def from_preset(
         cls,
         preset,
         **kwargs,
     ):
-        raise NotImplementedError
+        """Instantiate anDeBERTa tokenizer from preset vocabulary.
```
Comment (jbischof) [inline keras_nlp/models/f_net/f_net_tokenizer.py:100]: Update docstring

File: keras_nlp/models/f_net/f_net_tokenizer.py [hunk 3]
```diff
@@ -84,12 +88,53 @@ def __init__(self, proto, **kwargs):
...
+
+        Args:
+            preset: string. Must be one of {{names}}.
+
+        Examples:
+        ```python
+        # Load a preset tokenizer.
+        tokenizer = keras_nlp.models.DebertaV3Tokenizer.from_preset(
```
Comment (jbischof) [inline keras_nlp/models/f_net/f_net_tokenizer.py:108]: Update docstring

**Review by jbischof — CHANGES_REQUESTED** [review]
Looks great, just some nits!

File: keras_nlp/models/f_net/f_net_preprocessor.py [hunk 4]
```diff
@@ -177,3 +181,19 @@ def call(self, x, y=None, sample_weight=None):
...
+    @classproperty
+    def presets(cls):
+        return copy.deepcopy(backbone_presets)
+
+    @classmethod
+    def from_preset(cls, preset, **kwargs):
+        return super().from_preset(preset, **kwargs)
+
+
+FNetPreprocessor.from_preset.__func__.__doc__ = Preprocessor.from_preset.__doc__
```
Comment (mattdangerw) [inline keras_nlp/models/f_net/f_net_preprocessor.py:194]: this can be removed now, and the from_preset redefinition

**Review by mattdangerw — APPROVED** [review]
Thank you!

**Review by jbischof — COMMENTED** [review]
Thanks!

**Pull request merged**